packages/java-spring/src/spring/components/annotations.tsx

```diff
@@ -0,0 +1,32 @@
+import * as jv from "@alloy-js/java"
```

Comment:
I think rename this file to something like `route-annotations.tsx`. And call the component something like `SpringRouteAnnotation` to be more specific as these are only for the service endpoints, naming it SpringAnnotation is too generic.